packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ConfirmPlan.tsx

```diff
@@ -299,7 +337,7 @@ const ConfirmPlan = ({
           <Button
             variant="primary"
             loading={loadingChangeSubscription}
```

Comment:
I'd add it here as well, wdyt?
```suggestion
            loading={loadingChangeSubscription || !checkSubscriptionUpdate}
```